grommet/grommet
# Fix duplicate story names

#### What does this PR do?
Fixes duplicated storybook names. In some cases, the stories had been extracted to separate files, but the combined file still remained. In other cases, the Typescript names were duplicating the js story names.

#### Where should the reviewer start?
Any of the files.

#### What testing has been done on this PR?
The console warnings were cleared after these changes.

#### How should this be manually tested?
Check the console to make sure there are no warnings saying "story already exists in the store..."

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)
<img width="377" alt="Screen Shot 2019-12-12 at 2 50 22 PM" src="https://user-images.githubusercontent.com/12522275/70756946-c7ebbc80-1cf2-11ea-9f28-13ead5023255.png">

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
No.

#### Is this change backwards compatible or is it a breaking change?
Yes.
